golem/managers/proposal/plugins/buffer.py

```diff
@@ -65,7 +69,21 @@ async def _on_added(self, proposal: Proposal) -> None:
 
     async def _on_expired(self, proposal: Proposal):
         logger.debug("Rejecting expired `%r` and requesting fill", proposal)
-        await proposal.reject("Proposal no longer needed due to its near expiration.")
+
+        try:
+            await proposal.reject("Proposal no longer needed due to its near expiration.")
+        except ApiException as e:
+            message = json.loads(e.body)["message"]
+            if e.status == 400 and re.match(
+                r"^Subscription \[([^]]+)\] (wasn't found|expired).$", message
+            ):
+                logger.warning(
+                    "Manual proposal expiration seems to be called too late."
+                    " Consider shortening the expire duration."
+                )
+            else:
+                raise
```

Comment:
Why would we want to rise other `400`?

Reply:
I wanted to approach this as selective as possible to mitigate other unknown error cases to be handled the same as specifically tested ones for timing issues on expiration.